### PR TITLE
Add remote execution example payload

### PR DESCRIPTION
**Pull request by lzap**
Just an example to show what are the possible fields. Tested with safemode on.

### CODE REVIEW

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<%#
+name: Remote Execution Host Job
+description: Example payload for actions.remote_execution.run_host_job_succeeded
+snippet: false
+model: WebhookTemplate
+-%>
+#
+# Example webhook template, turn this into a XML, JSON, shell script etc.
```
Comment (ofedoren) [inline app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:8]: In safe mode there is only JSON available through `payload()` macro. Should we re-write the template to more explicit ready to use version/example?

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# For more information about available helpers visit /templates_doc
+#
+# Run job host name <%= @object.host_name %>
```
Comment (ofedoren) [inline app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:11]: Probably we should mention that `@object` here means `Action`, not `Task`.

**Review by ofedoren — COMMENTED** [review]
Thanks, @lzap, this is quite useful!

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb (hunk 1 above)
Comment (lzap) [reply app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb]: I dropped mentioning formats completely.

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb (hunk 2 above)
Comment (lzap) [reply app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:11]: Done!

**Review by ofedoren — APPROVED** [review]
Thanks, @lzap! Merging.

**Pull request merged**